### DISCUSSION TITLE
Can we add Emmet commands to the editor?

### DISCUSSION THREAD

**Discussion by johanbove** [beakerbrowser/beaker#1660] [General] [answered]
I really enjoy how Emmet speeds up writing HTML in VS Code and other editors where it is supported. It would be very useful to also be able to use the Emmet chunks and commands in Beaker editor.

- Would it be possible to add Emmet to the Editor?
- Should this be user-configurable?

## Resources
- <https://docs.emmet.io/>

**Comment by pfrazee** [comment, marked as answer]
I feel like something like this should be in a plugin for the editor, which means the editor would need plugins.

**Reply by johanbove** [reply]
True. Emmet should be includable as a plugin or extension, some day.
